package/yast2-ruby-bindings.changes

```diff
@@ -1,4 +1,9 @@
 -------------------------------------------------------------------
+Wed Mar  8 12:55:38 UTC 2017 - jreidinger@suse.com
+
+- Do not crash when invalid path is passed (bsc#1028081)
```

Comment:
yes, I do not plan to release it now, only if another fix appear as it is minor and usually root of issue is elsewhere ( in given bug report problem is in network ).